extensions/librdkafka/PublishKafka.cpp

```diff
@@ -129,9 +131,16 @@ bool PublishKafka::configureNewConnection(const std::shared_ptr<KafkaConnection>
 
   auto key = conn->getKey();
 
+  if (context->getProperty(DebugContexts.getName(), value) && !value.empty()) {
+    rd_kafka_conf_set(conf_, "debug", value.c_str(), errstr, sizeof(errstr));
```

Comment:
When will we receive feedback regarding this setting?

Reply:
Can you explain it? I think I don't know what does "feedback about the setting" mean.

Reply:
If we can'd ultimately do allow able values ( i commented above ) -- then that means we need to rely on this CSV. Very little validation is performed here -- so I presume librdkafka does that validation. In those cases do you get an immediate response or must we wait until data is supplied and sent through Kafka?

Reply:
From what I saw on the logs, we can only know after sending the data, and librdkafka will terminate before the first invalid setting ("topic,broker,mewrong,msg" then only "topic,broker" are applied). For validation, I know that I can split the string by "," then check whether they are valid or not. If you have better idea, please let me know.